src/main/java/hudson/plugins/sloccount/SloccountBuildAction.java

```diff
@@ -101,11 +115,33 @@ SloccountBuildAction getPreviousAction(){
         return null;
     }
 
-    public AbstractBuild<?,?> getBuild(){
+    public Run<?,?> getBuild(){
         return this.build;
     }
 
     public Object getTarget() {
         return this.result;
     }
+    
+    @Override
+    public void onLoad(Run<?,?> r)
+    {
+        this.build = r;
+        if( this.projectActions == null )
+        {
+            this.projectActions = new ArrayList<>();  
```

Comment:
This contruction I add after lots of fights with NPE after Jenkins restart.
Normally constructor is called and list is created. Later on onAttached is called and ProjectAction is added.
However after Jenkins shutdown and restart build is retrieved from disk.
In that case constructor is not called and onLoad raised NPE when I tried to add action to it.
As a result Action is not displayed at main Job page.